src/app/core/layouts/header/header.component.ts

```diff
@@ -28,4 +73,6 @@ export class HeaderComponent {
       detail: this.$translateService.instant('auth.logout.success.detail'),
     });
   }
+
+  protected readonly encodeURI = encodeURI;
```

Comment:
What is this ?

Reply:
Thanks for the comments, but I still didn't touch my header since we couldn't solve the problem. I will clean this part, but there's still things to do and solve with the header.